### PR TITLE
Update old release removal template email

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/release/old-release-removal.md [hunk 1]
```diff
@@ -115,18 +115,24 @@ need, and download them from Koji.
...
->
->   Any sites running OSG &lt;3.X&gt; should upgrade to the current release series,
->   OSG &lt;3.Y&gt;.  If you need assistance upgrading, please contact us at
->   help@opensciencegrid.org.
+```
+On <DAYNAME, MONTH DAY>, the OSG will be removing the OSG <3.X> release
+series from our repositories.  This includes both RPMs and tarballs hosted
+on repo.opensciencegrid.org.
+
+All support for OSG <3.X> was discontinued at the end of <MONTH YEAR>.
```
Comment (osg-cat) [inline docs/release/old-release-removal.md]: As a reminder, support for OSG <3.x> ended after <MONTH, YEAR>.

File: docs/release/old-release-removal.md [hunk 2]
```diff
@@ -115,18 +115,24 @@ need, and download them from Koji.
...
 
 If we're dropping support for a distro (e.g. EL 5 when we drop OSG 3.2), add
 the following after the first paragraph:
 
->   Note that OSG &lt;3.X&gt; was the last release to support Enterprise Linux &lt;Z&gt;
->   distributions.
+```
+Note that OSG <3.X> was the last release to support Enterprise Linux <Z>
+distributions.
+```
```
Comment (osg-cat) [inline docs/release/old-release-removal.md]: [Suggestion, if you like:] Note that OSG <3.x> was the last OSG release series that supported Enterprise Linux version <z>. If you believe that you still need support for this operating system series, please contact us at ....

Comment (matyasselmeci) [reply docs/release/old-release-removal.md]: "If you believe that you still need support for this operating system series, please send a big fat check to Matyas Selmeci, 1210 West Dayton St ..."

**Comment by matyasselmeci** [comment]
Any other comments? If this is fine, I'll merge it and send it out.

**Pull request merged**